lab-kyle/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "lab-06-cowsay-http-server",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab-06-cowsay-http-server ======",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kyleschnirring/lab-06-cowsay-http-server.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/kyleschnirring/lab-06-cowsay-http-server/issues"
+  },
+  "homepage": "https://github.com/kyleschnirring/lab-06-cowsay-http-server#readme",
+  "devDependencies": {
+    "cowsay": "^1.1.6",
```

Comment:
'cowsay' is a dependency, not a devDependency.  Your server cannot run without it.